release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -601,6 +601,13 @@ With this release, all `etcd` certificates originate from a new namespace: `open
 
 Manual rotation of signer certificates is still supported by deleting the specific secret and waiting for the status pod rollout to complete.
 
+[id="ocp-17-sigstore-verification_{context}"]
+==== Sigstore signature image verification
+
+With this release, Technology Preview clusters use Sigstore signatures to verify images that were retrieved using a pull spec that references `quay.io/openshift-release-dev/ocp-release`.
+
+Currently, if you are mirroring images, you must also mirror `quay.io/openshift-release-dev/ocp-release:<release_image_digest_with_dash>.sig` Sigstore signatures to your cluster in order for the image verification to succeed.
```

Comment:
Maybe drop "to your cluster"?  You're mirroring to the mirror, and then the cluster is retrieving from the mirror, but "you must also mirror ... to the mirror" seems redundant, and "you must also mirror ..." already implies "to the mirror".